osbuild/pipeline.py

```diff
@@ -282,20 +281,18 @@ def build_stages(self, object_store, interactive, libdir, secrets):
             if stage.checkpoint:
                 object_store.commit(tree, stage.id)
 
-        results["tree_id"] = self.tree_id
         return results, build_tree, tree
 
-    def assemble(self, object_store, build_tree, tree, interactive, libdir):
+    def copy_output(self, path, output_directory):
+        if output_directory:
+            subprocess.run(["cp", "--reflink=auto", "-a", f"{path}/.", output_directory], check=True)
+
```

Comment:
Can be a `staticmethod`. How about moving that to `ObjectStore` as `copy_object` or something, since all uses of it are via `object_store.resolve_ref(self.output_id)`?

Reply:
True, moving it to the ObjectStore sounds good!